content/docs/about/policies/listeneroption.md

```diff
@@ -38,7 +38,7 @@ spec:
 
 ### Option 2: Attach the policy to a particular listener on the gateway (`targetRefs.sectionName`)
 
-Instead of attaching a policy to all the listeners that are defined on the gateway, you can target a particular listener by using the `spec.targetRefs.sectionName` field in the ListenerOption resource. 
+Instead of attaching a policy to all the listeners that are defined on the gateway, you can target a particular listener by using the `spec.targetRefs.sectionName` field in the ListenerPolicy resource. 
```

Comment:
```suggestion
Instead of attaching a policy to all the listeners that are defined on the gateway, you can target a particular listener by using the `spec.targetRef.sectionName` field in the ListenerPolicy resource. 
```